docs/workflow/requirements/windows-requirements.md

```diff
@@ -42,7 +42,7 @@ The dotnet/runtime repository requires at least Visual Studio 2019 16.6.
 - Add its location (e.g. C:\Program Files (x86)\CMake\bin) to the PATH environment variable.
   The installation script has a check box to do this, but you can do it yourself after the fact following the instructions at [Adding to the Default PATH variable](#adding-to-the-default-path-variable).
 
-The dotnet/runtime repository recommends using CMake 3.16.0 or newer, but works with CMake 3.15.5.
+The dotnet/runtime repository recommends using CMake 3.16.4 or newer, but it may work with CMake 3.15.5.
```

Comment:
3.16.4 is also the version we have in https://github.com/dotnet/runtime/blob/main/global.json#L11